src/content/docs/en/recipes/sharing-state.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Share State Between Astro Components
+description: Learn how to share state across Astro components — with Nano Stores.
+i18nReady: true
+type: recipe
+---
+
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+:::tip
+Using Framework components? See [how to share state between Islands](/en/core-concepts/sharing-state/)!
```

Comment:
Nit: `Framework components` -> `framework components` for consistency with other pages
```suggestion
Using framework components? See [how to share state between Islands](/en/core-concepts/sharing-state/)!
```

Reply:
good catch!